### DISCUSSION TITLE
Issue Fully Customizing Tremors DonutChat and Progress Circle

### DISCUSSION THREAD

**Discussion by abdullah43577** [tremorlabs/tremor-npm#906] [Q&A]
I incorporated tremor's library in one of my projects and decided to use the DonutChat and Progress Circle, I'm trying to achieve something like the images attache below
![Screenshot 2024-01-14 001112](https://github.com/tremorlabs/tremor/assets/100965547/7069c9fd-939b-46c3-8d52-5c2480c40be1)
![Screenshot 2024-01-14 001122](https://github.com/tremorlabs/tremor/assets/100965547/168a90de-eb07-4c47-90f4-caab36fa8466)

and here's what I've achieved  using it so far
![Screenshot 2024-01-14 001151](https://github.com/tremorlabs/tremor/assets/100965547/39a86388-0cbb-453e-b230-9735945cfd17)
![Screenshot 2024-01-14 001158](https://github.com/tremorlabs/tremor/assets/100965547/58b5b8b0-61c3-48a8-927e-41c6cb7d2b5a)

even if the first image of the second design I posted looked a little similar, it's still not there yet, the documentation didn't allow for modifying some crucial parts of the elements to achieve something closer to those images. 

Take for example, the donutchart, I couldn't add my own specific colours, or at least I haven't figured out how to add my own, also I can't increase the strokeWidth of the donutChart to look big just like the first image I sent, this includes the text within the donutChart, I can't also edit the styles of that text.

I'd be glad if anyone could explain how I'm supposed to achieve this using the tremor library.

here's a piece of my code for reference

``` 
const customColors: (Color | string)[] = ['#7591F6', '#5E3FBE', '#31C1FF'];

 <DonutChart className="mt-[88px]" data={cities} category="sales" index="name" valueFormatter={valueFormatter} colors={customColors} showAnimation={true} animationDuration={2000} label="48 B" />
```

**Comment by hydrohiester** [comment]
As far as I can tell, you have to add those hex colors to your tailwind.config.js as documented here,  https://www.tremor.so/docs/getting-started/theming#custom-charts-colors

**Comment by severinlandolt** [comment]
Hi! In your case, I would switch to our copy-and-paste components: https://tremor.so/docs/visualizations/donut-chart and https://tremor.so/docs/visualizations/progress-circle
